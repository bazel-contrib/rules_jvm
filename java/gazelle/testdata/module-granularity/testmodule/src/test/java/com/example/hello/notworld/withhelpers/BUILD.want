load("@contrib_rules_jvm//java:defs.bzl", "java_test_suite")

java_test_suite(
    name = "withhelpers-tests",
    srcs = [
        "Helper.java",
        "NotWorldWithHelpersTest.java",
        "withdirectory/AnotherTest.java",
        "withdirectory/Helper.java",
    ],
    _gazelle_imports = [
        "com.example.hello.notworld.withhelpers",
        "com.example.hello.notworld.withhelpers.withdirectory",
        "java.lang.String",
        "java.lang.String", # TODO: Why is there a duplicate?
        "org.hamcrest.MatcherAssert",
        "org.hamcrest.Matchers",
        "org.junit.Assert",
        "org.junit.Test",
        "org.junit.jupiter.api.Test",
    ],
    _java_packages = [
        "com.example.hello.notworld.withhelpers",
        "com.example.hello.notworld.withhelpers.withdirectory",
    ],
    runner = "junit5",
    runtime_deps = [
        "@maven//:org_junit_jupiter_junit_jupiter_engine",
        "@maven//:org_junit_platform_junit_platform_launcher",
        "@maven//:org_junit_platform_junit_platform_reporting",
    ],
)
