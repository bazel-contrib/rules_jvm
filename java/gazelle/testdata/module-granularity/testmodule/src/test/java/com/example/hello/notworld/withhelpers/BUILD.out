load("@contrib_rules_jvm//java:defs.bzl", "java_test_suite")

# gazelle:java_module_granularity module

java_test_suite(
    name = "withhelpers-tests",
    srcs = [
        "Helper.java",
        "NotWorldWithHelpersTest.java",
        "withdirectory/AnotherTest.java",
        "withdirectory/Helper.java",
    ],
    runner = "junit5",
    runtime_deps = [
        "@maven//:org_junit_jupiter_junit_jupiter_engine",
        "@maven//:org_junit_platform_junit_platform_launcher",
        "@maven//:org_junit_platform_junit_platform_reporting",
        "@maven//:org_junit_vintage_junit_vintage_engine",
    ],
    deps = [
        "@maven//:junit_junit",
        "@maven//:org_hamcrest_hamcrest_all",
        "@maven//:org_junit_jupiter_junit_jupiter_api",
    ],
)
